LazyCache/LazyCacheOptions.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace LazyCache
+{
+    public class LazyCacheOptions
+    {
+        public int DefaultCacheDurationSeconds { get; set; } = 60 * 20;
+        public int NumberOfKeyLocks { get; set; } = Math.Max(Environment.ProcessorCount * 8, 32);
+
+    }
+}
```

Comment:
Including an empty line at the end of a file ensures consistency in version control diffs.